bin/node_gc

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
```

Comment:
Well, *I* do, so until we have a style guide for shell scripts, I'm leaving it. :stuck_out_tongue_closed_eyes: